Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs

```diff
@@ -117,7 +117,9 @@ public bool IsActiveBuildTargetValid()
             { Configurations.LatestScriptingRuntime, new ConfigGetter(IsLatestScriptingRuntime) },
             { Configurations.ForceTextSerialization, new ConfigGetter(IsForceTextSerialization) },
             { Configurations.VisibleMetaFiles, new ConfigGetter(IsVisibleMetaFiles) },
+#if !UNITY_2019_3_OR_NEWER
             { Configurations.VirtualRealitySupported, new ConfigGetter(() => XRSettingsUtilities.LegacyXREnabled) },
```

Comment:
I still think we might have a bug here if we don't port the `if` over though, since this will still happen while the checkbox isn't visible if `if (!MixedRealityOptimizeUtils.IsBuildTargetAndroid() && !MixedRealityOptimizeUtils.IsBuildTargetIOS() && XRSettingsUtilities.IsLegacyXRActive)` returns `false.

Reply:
i will double check, but i believe the recent commit has resolved this concern.

Reply:
In fact, i tested this on a project without xrsdk enabled and not only did the check box not appear, it also did not reproduce the bug.

Reply:
or are you saying we may still need the checkbox for Android / iOS?